test/built-ins/Array/prototype/toReversed/frozen-this-value.js

```diff
@@ -1,3 +1,4 @@
+// |reftest| shell-option(--enable-change-array-by-copy) skip-if(!this.hasOwnProperty('getBuildConfiguration')||!getBuildConfiguration()['change-array-by-copy'])
```

Comment:
Yes, I think there's also a manifest file that the tests can be added to. I'll take a look.

Reply:
I added a patch to my stack for SpiderMonkey so that the test262-update.py script will add these prefixes to the tests automatically: https://phabricator.services.mozilla.com/D142748 . So when test262 updates, SpiderMonkey will already know how to run the tests.

So now this pull request only includes two changes: the additions to `length-exceeding-array-length-limit.js` and, and adding `features: [change-array-by-copy]` to one of the tests where it was missing (this is so that the test262-update.py script will work properly.)

Reply:
Awesome, thanks!